src/vscode-bicep-ui/.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
```

Comment:
Could we enforce this at the repo level instead?

Reply:
Good point. Merged this and other `.editorconfig` files in sub-folders into the root config file.